Resources/Prototypes/_Goobstation/Voice/speech_emotes.yml

```
@@ -215,4 +217,20 @@
+    - wurbled
+
+- type: emote
+  id: Gulp
+  name: chat-emote-name-gulp
+  category: Vocal
+  icon: _Goobstation/Interface/Emotes/gulp.png
+  whitelist:
+    components:
+    - Vocal
+  blacklist:
+   components:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 3

(indentation)